lib/game_message_generator.rb

```diff
@@ -27,4 +27,14 @@ def game_over(params)
   def previous_move(params)
     "Previous Move: #{params[:player]} on #{params[:position]}"
   end
+
+  def match_up(params)
+    player_one = params[:players][0]
+    player_two = params[:players][1]
+
+    player_one_type = player_one.class.name.gsub('Player', '')
```

Comment:
This is neat, but feels overly complicated and couples the class names to the UI.